command/certificate/install.go

```diff
@@ -17,50 +17,50 @@ func installCommand() cli.Command {
 	return cli.Command{
 		Name:   "install",
 		Action: command.ActionFunc(installAction),
-		Usage:  "install a root certificate in the system truststore",
+		Usage:  "install a root certificate in the supported trust stores",
 		UsageText: `**step certificate install** <crt-file>
 [**--prefix**=<name>] [**--all**]
 [**--java**] [**--firefox**] [**--no-system**]`,
-		Description: `**step certificate install** installs a root certificate in the system
-truststore.
+		Description: `**step certificate install** installs a root certificate in
+the supported trust stores.
 
-Java and Firefox truststores are also supported via the respective flags.
+Java's and Firefox's trust stores are also supported via the respective flags
 
 ## POSITIONAL ARGUMENTS
 
 <crt-file>
-:  Certificate to install in the system truststore
+:  Root certificate to install in the specified trust stores.
 
 ## EXAMPLES
 
-Install a certificate in the system truststore:
+Install a root certificate in the system's default trust store:
 '''
 $ step certificate install root-ca.pem
 '''
 
-Install a certificate in all the supported truststores:
+Install a root certificate in all the supported trust stores:
 '''
 $ step certificate install --all root-ca.pem
 '''
 
-Install a certificate in Firefox and the system truststore:
+Install a root certificate in Firefox's and the system's default trust store:
 '''
 $ step certificate install --firefox root--ca.pem
```

Comment:
Just noticed this extra dash

```suggestion
$ step certificate install --firefox root-ca.pem
```

Reply:
Just pushed a commit to fix this typo as well. Sharp eyes!